server/base/src/test/java/org/apache/accumulo/server/master/balancer/TableLoadBalancerTest.java

```diff
@@ -146,9 +146,7 @@ public void test() {
     TableConfiguration conf = createMock(TableConfiguration.class);
     // Eclipse might show @SuppressWarnings("removal") as unnecessary.
     // Eclipse is wrong. See https://bugs.eclipse.org/bugs/show_bug.cgi?id=565271
-    @SuppressWarnings("removal")
-    Property TABLE_CLASSPATH = Property.TABLE_CLASSPATH;
-    expect(conf.resolve(Property.TABLE_CLASSLOADER_CONTEXT, TABLE_CLASSPATH))
+    expect(conf.resolve(Property.TABLE_CLASSLOADER_CONTEXT))
```

Comment:
This would be a weird thing to expect. Probably can be removed when the code that we're expecting is updated to avoid the resolution of a single non-deprecated property, and just uses that property directly.